Project.toml

```diff
@@ -3,8 +3,8 @@ uuid = "08131aa3-fb12-5dee-8b74-c09406e224a2"
 version = "0.8.1"
 
 [deps]
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+OpenCL_jll = "6cb37087-e8b6-5417-8430-1f242f1e46e4"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 
```

Comment:
I think we also need to add a [compat] section

Reply:
would be like this ?
```
[compat]
OpenCL_jll = "2022.9.23"
```

Reply:
Yes, I think that is right. Can you update the PR so that I can merge it?